app.config.ts

```diff
@@ -1,15 +1,21 @@
 import {ExpoConfig, ConfigContext} from '@expo/config';
 
-export default ({config}: ConfigContext): ExpoConfig => {
```

Comment:
What kind of PR check would have caught this? Surprised the build didn't barf ...

Reply:
I just ran `yarn tsc` at the root to see what the Typescript compiler had to say. I'm surprised that the build didn't barf too! 

This seems to be by "design"? 

https://stackoverflow.com/questions/58459262/why-does-my-react-native-app-build-successfully-despite-typescript-compiler-erro